go/obscuronode/enclave/enclave.go

```diff
@@ -419,18 +450,28 @@ type processingEnvironment struct {
 	processedTxsMap map[common.Hash]nodecommon.L2Tx
 }
 
-func NewEnclave(id common.Address, mining bool, collector StatsCollector) nodecommon.Enclave {
+// NewEnclave creates a new enclave.
+// `genesisJSON` is the configuration for the corresponding L1's genesis block. This is used to validate the blocks
```

Comment:
the genesisJson can't be passed in by the L1 host. It will have to be hardcoded in a config file bundled  n the sgx image and signed.

Reply:
yes